src/main/java/duke/task/DeadlineTask.java

```diff
@@ -0,0 +1,46 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duke.exception.DukeException;
+
+/**
+ * Represents a deadline task.
+ */
+public class DeadlineTask extends Task {
+    /**
+     * Deadline of task.
+     */
+    private LocalDateTime deadline;
```

Comment:
JavaDocs for private attributes and methods are optional. But it is still good to include them. 👍